frame/multisig/src/lib.rs

```diff
@@ -122,6 +124,8 @@ decl_storage! {
 		pub Multisigs: double_map
 			hasher(twox_64_concat) T::AccountId, hasher(blake2_128_concat) [u8; 32]
 			=> Option<Multisig<T::BlockNumber, BalanceOf<T>, T::AccountId>>;
+
+		pub Calls: map hasher(identity) [u8; 32] => Option<(Vec<u8>, T::AccountId, BalanceOf<T>)>;
```

Comment:
Nit: maybe add descriptive type defs for `[u8; 32]` (`CallHash`?) and `Vec<u8>` (`EncodedCall`) ?